### PR TITLE
Communication R4 base (read, search, and write)

### PR DESCRIPTION
**Pull request by VelocityKnight**
Description
----
Add mocked Communication read, search, and write to fhir.cerner.com. Will not be merged until full, non-mocked Communication workflow is in production.

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.
- [ ] Full, non-mocked Communication in production.

Screenshots
---
image
image

### CODE REVIEW

File: content/millennium/r4/request-and-response/communication.md [hunk 1]
```diff
@@ -61,13 +61,13 @@ _Implementation Notes_
...
- `category`   | This or `_id` or `recipient` or `received`      | [`token`]     | A token for a [`CodeableConcept`] that points to the CodeableConcept used for Communication.category. Example: `category=http://terminology.hl7.org/CodeSystem/communication-category|notification` or `category=http://terminology.hl7.org/CodeSystem/communication-category|reminder`
- `recipient`  | This or `_id` or `category`  or `received`      | [`reference`] | The recipient of the Communication. Example: `recipient=Practitioner/3456783` or `recipient:Group=8765556`
- `received`   | This or `_id` or `category`  or `recipient`     | [`date`]      | Date range into which the communication falls. Example: `received=le2017-02-01T10:30:00Z` or `received=ge2020-03-16T21:31:35.458Z&received=le2020-07-16T21:31:35.457Z"`
+ `category`   | This or `_id` or `recipient` or `received`      | [`token`]     | A token for a [`CodeableConcept`] that points to the CodeableConcept used for the communication.category. Example: `http://terminology.hl7.org/CodeSystem/communication-category|notification`
```
Comment (ebaranowski) [inline content/millennium/r4/request-and-response/communication.md:64]: This shouldnt be changed. It's a field path.

Comment (VelocityKnight) [reply content/millennium/r4/request-and-response/communication.md]: Reverted: https://github.com/cerner/fhir.cerner.com/pull/440/commits/cf002fdb54739e842215998ff3ed7e977152dc2b and updated screenshot

**Comment by ebaranowski** [comment]
Looks good. done reviewing

**Comment by VelocityKnight** [comment]
Closed, code has been reviewed. Will reopen when communication is allowlisted in prod and a full service (local service also in prod, not mocked).

**Pull request closed without merging**